docs/user_guide/hopsworks/featurestore.rst

```diff
@@ -0,0 +1,609 @@
+
+.. _hopsworks_featurestore_create_fg1.png: ../../_images/feature_store/create_fg1.png
+.. figure:: ../../imgs/feature_store/create_fg1.png
+    :alt: Creating a new feature group with metadata from the feature registry UI.
+    :target: `hopsworks_featurestore_create_fg1.png`_
+    :align: center
+    :scale: 55 %
+    :figclass: align-center
+
+    Creating a new feature group with metadata from the feature registry UI.
+
+.. _hopsworks_featurestore_create_fg2.png: ../../_images/feature_store/create_fg2.png
+.. figure:: ../../imgs/feature_store/create_fg2.png
+    :alt: Creating a new feature group with metadata from the feature registry UI.
+    :target: `hopsworks_featurestore_create_fg2.png`_
+    :align: center
+    :scale: 55 %
+    :figclass: align-center
+
+    Creating a new feature group with metadata from the feature registry UI.
+
+* Updating metadata of a feature group using the UI registry:
+
+.. _hopsworks_featurestore_edit_fg1.png: ../../_images/feature_store/edit_fg1.png
+.. figure:: ../../imgs/feature_store/edit_fg1.png
+    :alt: Creating a new feature group with metadata from the UI registry.
+    :target: `hopsworks_featurestore_edit_fg1.png`_
+    :align: center
+    :scale: 55 %
+    :figclass: align-center
+
+    Editing the metadata of a feature from the feature registry UI.
+
+.. _hopsworks_featurestore_edit_fg2.png: ../../_images/feature_store/edit_fg2.png
+.. figure:: ../../imgs/feature_store/edit_fg2.png
+    :alt: Creating a new feature group with metadata from the UI registry.
+    :target: `hopsworks_featurestore_edit_fg2.png`_
+    :align: center
+    :scale: 55 %
+    :figclass: align-center
+
+    Editing the metadata of a feature from the feature registry UI.
+
+Other actions available in the feature registry are:
+
+* Create/Update/Delete operations on feature groups and training datasets
+* Preview feature group data
+* View feature group and training dataset schemas
+
+A Multi-tenant Feature Store Service
+------------
+
+Despite the benefit of centralizing features, we have identified a need to enforce access control to features. Several organizations that we have talked to are working partially with sensitive data that requires specific access rights that is not granted to everyone in the organization. For example, it might not be feasible to publish features that are extracted from sensitive data to a feature store that is public within the organization.
+
+To solve this problem we utilize the multi-tenancy property built-in to the architecture of the Hopsworks platform. Feature stores in Hopsworks are by default project-private and can be shared across projects, which means that an organization can combine public and private feature stores. An organization can have a central public feature store that is shared with everyone in the organization as well as private feature stores containing features of sensitive nature that are only accessible by users with the appropriate permissions.
```

Comment:
@Limmen `utilize the multi-tenant model of Hopsworks`,

Reply:
fixed